spec/models/service_template_provision_task_spec.rb

```diff
@@ -3,16 +3,16 @@
 describe ServiceTemplateProvisionTask do
   context "with multiple tasks" do
     before(:each) do
-      admin      = FactoryGirl.create(:user_admin)
+      @admin      = FactoryGirl.create(:user_with_group)
```

Comment:
Use a let block, then you do not have to change any of the other lines that already reference `admin`.